tests/onnx/test_model_transformer.py

```diff
@@ -139,3 +140,27 @@ def test_output_insertion(target_layers, target_layer_outputs):
     # Should be topologically sorted
     for i in range(len(target_layers)):
         assert onnx_graph.get_model_outputs()[i].name in target_layer_outputs
+
+CONV_LAYERS = [['Conv1', 'Conv2']]
+BIAS_VALUES = [[np.full((32,), 2), np.full((10,), 3)]]
+BIAS_REFERENCES = [[2.0, 3.0]]
+
+@pytest.mark.parametrize('layers, values, refs', zip(CONV_LAYERS, BIAS_VALUES, BIAS_REFERENCES))
+def test_bias_correction(layers, values, refs):
```

Comment:
Write new tests with type hints for parameters, it never hurts.

Reply:
But the existing tests don't have this type hints 👍